gensim/models/word2vec.py

```diff
@@ -1120,7 +1135,8 @@ def add_word(word, weights):
                     weights = fromstring(fin.read(binary_len), dtype=REAL)
                     add_word(word, weights)
             else:
-                for line_no, line in enumerate(fin):
+                for line_no in xrange(vocab_size):
```

Comment:
Yes; I meant why the line split.

`islice` seems more idiomatic and also safer (in case `vocab_size > len(fin)`).